e2e-tests/kuttl/scheduled-backup/05-update-cluster.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: pxc.percona.com/v1-9-0
```

Comment:
This should probably be a variable depending on which current version we are testing?

Reply:
done via env variables and yq